### ISSUE TITLE
malformed theme file contrast values when using themes.skeleton.dev

### ISSUE THREAD

**Issue by TazorDE** [skeletonlabs/skeleton#3402]
### Current Behavior

When setting `white` or `black` as the color contrast values they are added as malformed color values to the theme file.

```css
--color-primary-contrast-dark: 0 0 0;
--color-primary-contrast-light: 255 255 255;
```

![Image](https://github.com/user-attachments/assets/f4bd06af-9864-426a-b124-bb689dc73f63)

### Expected Behavior

They should be added as valid CSS colors

```css
--color-primary-contrast-dark: rgb(0 0 0);
--color-primary-contrast-light: rgb(255 255 255);
```
or

```css
--color-primary-contrast-dark: black;
--color-primary-contrast-light: white;
```

### Steps To Reproduce

1. Using the theme generator
2. Set the contrast colors to either `white` or `black`
3. They are added to the theme-file in a malformed manner

### Link to Reproduction / Stackblitz

themes.skeleton.dev

**Comment by endigo9740** [comment]
Let's make sure the background colors are updated as well. I believe those may have been overlooked as well.

**Comment by phamduylong** [comment]
I'll take a look

**Comment by endigo9740** [comment]
@phamduylong will you do me a favor and scan through our preset themes too. I think there may be a couple instances still using the old `XX XX XX` RGB values. Those will need to be updated too.

I you're using VS Code, I recommend this extension for converting the colors. You'll need to adjust to valid RGB format first but doing two things:

1. Wrap the color with `rgb()`
2. Add commas between the numbers
3. Then run this extension:

https://marketplace.visualstudio.com/items?itemName=starbist.oklchanger
